templates/telemetry/pioneer-study/pioneer-study.4.schema.json

```diff
@@ -0,0 +1,67 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "properties": {
+    @TELEMETRY_CREATIONDATE_1_JSON@,
+    @TELEMETRY_ID_1_JSON@,
```

Comment:
You will want to pull in the required application section here     @TELEMETRY_APPLICATION_1_JSON@,  Also update the validation input